src/index.html

```diff
@@ -2,422 +2,480 @@
-          </a>
-        </div>
-        <h3 class="text-center mb-4">Partners</h3>
-        <div class="col-10 d-flex flex-wrap justify-content-center mx-auto align-items-center">
-          <a href="https://www.yomuchacho.com" id="muchachos" target="_blank" rel="noopener">
-            <img class="px-3 logo" src="/assets/img/partners/muchachos.webp" alt="Muchachos">
-          </a>
-          <a href="https://www.launchleadership.org" id="launchleadership" target="_blank" rel="noopener">
-            <img class="px-3 logo" src="/assets/img/partners/launch.png" alt="Launch Leadership">
-          </a>
-          <a href="https://www.fusecoworking.com" id="fusecoworking" target="_blank" rel="noopener">
-            <img class="px-3 logo" src="/assets/img/partners/fuse.png" alt="FUSE Coworking">
-          </a>
-          <a href="http://makeshiftlincoln.org" id="makeshiftlincoln" target="_blank" rel="noopener">
-            <img class="px-3 logo" src="/assets/img/partners/makeshift.png" alt="Makeshift Lincoln">
-          </a>
-          <a href="https://www.selfimagedefined.org" id="selfimagedefined" target="_blank" rel="noopener">
-            <img class="px-3 logo" src="/assets/img/partners/selfid.png" alt="Self Image Defined">
-          </a>
-          <a href="https://leturaidigima.com" id="leturaidigima" target="_blank" rel="noopener">
-            <img class="px-3 logo" src="/assets/img/partners/letura.png" alt="Letura Idigima">
-          </a>
-          <a href="https://www.carlosvelascodesign.com" id="carlosvelascodesign" target="_blank" rel="noopener">
-            <img class="px-3 logo" src="/assets/img/partners/carlos.png" alt="Carlos Velasco Design">
-          </a>
-        </div>
-        <div class="col-10 mx-auto"><hr class="hr"></div>
-        <div class="mx-auto mt-5 d-flex justify-content-center align-items-center">
-          <a class="btn info-btn text-light" role="button"
-            href="assets/pdf/GCL-Donation-Info.pdf" id="donatebutton" target="_blank" rel="noopener">
-            Want to become a Sponsor or Partner?
-          </a>
-        </div>
-      </section>
+        <section class="bg-pink col-12 py-5" id="our-clubs">
+            <div class="text-center col-lg-7 col-xs-8 mx-auto mb-3">
+                <h2 class="font-weight-bold">Our Clubs</h2>
+                <i>
+                    Our clubs give girls interested in coding a gentle introduction to software, hardware,
+                    computer science fundamentals, and leadership through projects, activities, and community speakers.
+                </i>
+            </div>
+            <div class="col-12 d-flex justify-content-center">
+                <div class="col-lg-8 col-xs-10">
+                    <ul class="timeline">
+                        <li>
+                            <h5 class="font-weight-bold">Spring 2020</h5>
+                            <p>Three clubs will be held in Spring 2020 - a beginner club, a web development club, and a
```

Comment:
We should be sure to update this. It's already fall 2021 now whoa

Reply:
ope good catch! I'll switch it into past tense for now, but think of a better way to update in the future - continuing a point for each semester might get redundant